strings.go

```diff
@@ -7,3 +7,11 @@ func ReverseString(str string) string {
 	}
 	return string(byteString)
 }
+
+func CharactersCount(str string) int {
```

Comment:
What about just use len() here instead of loop?)

Reply:
len() returns number of bytes in the string but characters may be longer than 1 byte. Then I can use utf8.RunesCountInString but I thought that I should implement function by myself, anyways, this task is suspiciously simple